scripts/release/utils.js

```diff
@@ -49,8 +57,7 @@ const getArtifactsList = async buildID => {
     process.env.CIRCLE_CI_API_TOKEN
   }`;
   const workflowMetadata = await http.get(workflowMetadataURL, true);
-
-  const job = workflowMetadata.jobs.find(
+  const job = workflowMetadata.items.find(
```

Comment:
Not sure when this changed, but the metadata URL seems to now return `workflowMetadata.items` rather than `workflowMetadata.jobs`.